models/product/src/product-data/types.ts

```diff
@@ -13,6 +13,7 @@ import { ValueOf } from '@commercetools-test-data/core/src/@jackfranklin/test-da
 // The base generator model. Consumers configure these fields.
 export type TProductData = Omit<ProductData, 'categories'> & {
   categories: Array<Category>;
+  searchKeywords?: Array<SearchKeyword>;
```

Comment:
We shouldn't need to include this as it's already encapsulated in the `Omit<ProductData, 'categories'>` type

Reply:
Thanks Jai 🙇 

Removed here: dfc263c657ba6c4f9942f98fcc24e83379f9d381


Just out of curiosity. What do we need the `searchKeyword` property for?

Reply:
See [this comment](https://github.com/commercetools/test-data/pull/627#discussion_r1709482883), not sure if it got missed in resolution